hudi-common/src/main/java/org/apache/hudi/common/table/HoodieTableConfig.java

```diff
@@ -59,6 +60,8 @@
   public static final String HOODIE_TABLE_VERSION_PROP_NAME = "hoodie.table.version";
   public static final String HOODIE_TABLE_PRECOMBINE_FIELD = "hoodie.table.precombine.field";
   public static final String HOODIE_TABLE_PARTITION_COLUMNS = "hoodie.table.partition.columns";
+  public static final String HOODIE_TABLE_ROWKEY_FIELDS = "hoodie.table.rowkey.fields";
+  public static final String HOODIE_TABLE_SCHEMA = "hoodie.table.schema";
```

Comment:
rename: `hoodie.table.create.schema` to be clear that this is the schema at creation.